packages/earth-map/src/components/collection/collection-editplaces/CollectionEditPlaces.tsx

```diff
@@ -65,6 +64,7 @@ export function CollectionEditPlaces(props: IProps) {
             loadFunction={(query) =>
               PlacesService.fetchPlaces({
                 ...query,
+                filter: ['type', '!=', 'Collection'].join(''),
```

Comment:
Don't allow collections to add other collections during creation.

We might want this in the future? Create collections from other collections.